App/components/inputs/QRScanner.tsx

```diff
@@ -81,7 +81,7 @@ const QRScanner: React.FC<Props> = ({ handleCodeScan }) => {
           }}
         >
           <CameraViewContainer portrait={portraitMode}>
-            <CredentialOfferSteps activeStep={1} />
+            {/* <CredentialOfferSteps activeStep={1} /> */}
```

Comment:
Remove dead code?

Reply:
Will do. Realized after the previous PR that the scanner is used for both Credential Offers and Proof Requests so the text added won't make sense. Will remove for now.